spanner/value.go

```diff
@@ -877,6 +877,100 @@ func (n *PGNumeric) UnmarshalJSON(payload []byte) error {
 	return nil
 }
 
+// NullProtoMessage represents a Cloud Spanner PROTO that may be NULL.
+type NullProtoMessage struct {
+	ProtoMessageVal proto.Message // ProtoMessageVal contains the value when Valid is true, and nil when NULL.
+	Valid           bool          // Valid is true if ProtoMessageVal is not NULL.
```

Comment:
We should add an extra comment here to explain to users what they need to do to write a null value. Normally, they would be able to set `Valid: false` to create a null value, but if I understand it correctly, that is not the case for proto columns.